src/tests/ftest/security/cont_acl.yaml

```diff
@@ -24,8 +24,8 @@ container:
   control_method: daos
 container_acl:
   acl_file_name: cont_test_acl1.txt
-  new_user: daos_ci_tester_1
-  new_group: daos_ci_test_grp_1
+  new_user: root
+  new_group: root
```

Comment:
Does this now assume that the user account is pre-existing?  If so, should we at least have a comment here?

Reply:
I think it's safe to assume root already exists